### ISSUE TITLE
search window does not work when entered japanese character

### ISSUE THREAD

**Issue by Seasawher** [lumeland/theme-simple-blog#14]
When I type in Japanese, nothing is found in the search. 

Can you support Japanese search or hide the search window?

**Comment by oscarotero** [comment]
Ok, I have to include the ability to customize the pagefind plugin (https://lume.land/plugins/pagefind/#the-binary-file)

**Comment by oscarotero** [comment]
I have released v0.10.0, and you can configure the pagefind plugin to support japanese characters:

```js
use.blog({
  pagefind: {
    binary: {
      extended: true
    },
  }
})

```
